api/src/main/java/org/openmrs/TestOrder.java

```diff
@@ -162,6 +162,8 @@ public TestOrder cloneForDiscontinuing() {
 		newOrder.setPreviousOrder(this);
 		newOrder.setPatient(getPatient());
 		newOrder.setOrderType(getOrderType());
+		newOrder.setOrderGroup(null);
```

Comment:
Shouldn't this be the orderGroup from the order being cloned? Otherwise it is redundant to be setting null because anyways thy are already null